src/assets/news-icon.js

```diff
@@ -1,21 +1,21 @@
-      <path d="M36 0H129C142.807 0 154 11.1929 154 25V100.199C154 122.29 136.091 140.199 114 140.199H36V0Z" fill="#F8F5E5"/>
-      <rect x="17" y="43.51" width="19" height="82.1854" fill="#F8F5E5"/>
-      <path d="M147.227 0H38.8636C37.0674 0 35.3447 0.820605 34.0746 2.28129C32.8045 3.74198 32.0909 5.72309 32.0909 7.78882V31.1553H11.7727C9.97649 31.1553 8.25382 31.9759 6.98369 33.4366C5.71355 34.8972 5 36.8784 5 38.9441V116.832C5 123.029 7.14066 128.973 10.9511 133.355C14.7615 137.737 19.9295 140.199 25.3182 140.199H133.682C139.071 140.199 144.239 137.737 148.049 133.355C151.859 128.973 154 123.029 154 116.832V7.78882C154 5.72309 153.286 3.74198 152.016 2.28129C150.746 0.820605 149.024 0 147.227 0ZM18.5455 116.832V46.7329H32.0909V116.832C32.0909 118.898 31.3774 120.879 30.1072 122.34C28.8371 123.8 27.1144 124.621 25.3182 124.621C23.5219 124.621 21.7993 123.8 20.5291 122.34C19.259 120.879 18.5455 118.898 18.5455 116.832ZM140.455 116.832C140.455 118.898 139.741 120.879 138.471 122.34C137.201 123.8 135.478 124.621 133.682 124.621H44.4173C45.2058 122.123 45.618 119.489 45.6364 116.832V15.5776H140.455V116.832Z" fill="#7BB3D2"/>
-      <rect x="61" y="99.5894" width="66" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="61" y="109.258" width="66" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="96" y="61.8809" width="31" height="30.9404" rx="3" fill="#7BB3D2"/>
-      <rect x="61" y="31.9073" width="66" height="13.5364" rx="2" fill="#7BB3D2"/>
-      <rect x="61" y="51.2451" width="66" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="61" y="60.9139" width="30" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="61" y="89.9205" width="30" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="61" y="80.2516" width="30" height="3.86755" rx="1.93378" fill="black"/>
-      <rect x="61" y="70.5828" width="30" height="3.86755" rx="1.93378" fill="black"/>
+    <path d="M23.8921 156.829H136.809C136.809 156.829 141.645 154.875 143.245 152.111C144.844 149.348 145 142.676 145 142.676V10H23.8921V47.7391M23.8921 156.829C23.8921 156.829 23.5562 157.814 15.7012 155.06C7.84654 152.305 4 142.676 4 142.676V47.7391H13.9461H23.8921M23.8921 156.829V47.7391" stroke="#6BB4D6" stroke-width="8" stroke-linejoin="round"/>
```

Comment:
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/33645465/92850744-cbfc5600-f3ba-11ea-9e94-c474644e208e.png">
 is this icon too big compared to the rest? If so can someone please help me w/ svg sizing

Reply:
i dont think it's too off but if you want it smaller just throw in a classname in the aboutpage and set a width. (same as the other svgs on this page)